packages-next/keystone/src/___internal-do-not-use-will-break-in-patch/admin-ui/pages/ListPage/SortSelection.tsx

```diff
@@ -43,7 +43,14 @@ export function SortSelection({ list }: { list: ListMeta }) {
         </span>
       </Button>
 
-      <PopoverDialog arrow={arrow} isVisible={isOpen} {...dialog.props} ref={dialog.ref}>
+      <PopoverDialog
+        aria-label="Sort options dialog"
```

Comment:
q: when a screen reader will announce this, won't they already say that this is a dialog because of other aria attributes or no?

Reply:
that's a good point,
```suggestion
        aria-label="Sort options"
```